frontend/src/app/login/login.component.html

```diff
@@ -1,16 +1,16 @@
 <h2>Logowanie</h2>
 <form method="post" role="form" (ngSubmit)=login()>
-    <div class="form-group">
-      <p>Email:</p>
-        <input type="text" [(ngModel)]="loginModel.username" name="username" class="form-control"/>
-    </div>
-    <div class="form-group">
-      <p>Password:</p>
-        <input type="password" [(ngModel)]="loginModel.password" name="password" class="form-control"/>
-    </div>
-    <div class="form-group">
-        <a [routerLink]="['/password-reset']">Nie pamiętasz hasła? Możemy pomóc!</a>
-    </div>
-    <div *ngIf="loginError$ | async" class="alert alert-danger" role="alert">Niepoprawne dane logowania</div>
-    <button type="submit" name="submit" class="btn btn-primary">Zaloguj</button>
+  <div class="form-group">
+    <p>Email:</p>
+    <input type="text" [(ngModel)]="loginModel.username" name="username" class="form-control" />
+  </div>
+  <div class="form-group">
+    <p>Password:</p>
```

Comment:
if You're doing some cleanup here, also switch label names from English to Polish, pleace